docs/variants/qemu_q35/firmware-update.md

```diff
@@ -1,16 +1,6 @@
 # Firmware Update
 
-The update process of Dasharo (UEFI) firmware on QEMU Q35 does not need any
-special actions. QEMU Q35 is an emulated platform, not real hardware.
-
-Updating the Dasharo OVMF firmware is as simple as invoking a QEMU command
-with a newer `OVMF*.fd` files:
-
-```bash
-qemu-system-x86_64 -machine q35,smm=on \
-	-global driver=cfi.pflash01,property=secure,value=on \
-	-drive if=pflash,format=raw,unit=0,file=Build/OvmfX64/RELEASE_GCC5/FV/OVMF_CODE.fd,readonly=on \
-	-drive if=pflash,format=raw,unit=1,file=Build/OvmfX64/RELEASE_GCC5/FV/OVMF_VARS.fd \
-	-debugcon file:debug.log -global isa-debugcon.iobase=0x402 \
-	-global ICH9-LPC.disable_s3=1
-```
+As of now, there is no internal update process in Dasharo (coreboot+UEFI) for
+QEMU Q35 platform, but it is on our wish-list. Until then, Dasharo firmware can
```

Comment:
Can we link to the issue with the update process request or one that will lead to implementing it?

Reply:
https://github.com/Dasharo/docs/compare/2a47bdd5f3eae81d35cc9545010489747bac76c6..fd1fc06fa12f4d9dde6a66b01a567657c6bb1a5d#diff-f9ce46bad7f8a8163eef79ce1acf0a799b3c481bfc469d986c564af001f0930eR4